frontend/web/src/app/components/customer-list/customer-list.component.ts

```diff
@@ -100,10 +101,10 @@ export class CustomerListComponent implements OnInit, AfterViewInit {
   }
 
   private prepareTriggers() {
-    /*let _self = this;
+    let _self = this;
     (<any>this.table).on('click-row.bs.table', function (row, $element) {
-      _self.router.navigate([_self.router.url, $element.id]);
-    })*/
+      _self.selectedCustomer =  _self.customers.filter(customer => {if(customer.id ==  $element.id)return customer})[0];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') == should be === [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1440)